plugins/contact-resources/src/components/EmployeeRefPresenter.svelte

```diff
@@ -3,49 +3,31 @@
   import { Ref } from '@hcengineering/core'
   import { ButtonKind } from '@hcengineering/ui'
   import { PersonLabelTooltip } from '..'
-  import contact from '../plugin'
-  import { employeeByIdStore } from '../utils'
-  import AssigneeBox from './AssigneeBox.svelte'
-  import EmployeePresenter from './EmployeePresenter.svelte'
+  import EmployeeAttributePresenter from './EmployeeAttributePresenter.svelte'
 
-  export let value: Ref<Employee> | null | undefined
+  export let value: Ref<Employee> | Ref<Employee>[] | null | undefined
   export let kind: ButtonKind = 'link'
   export let tooltipLabels: PersonLabelTooltip | undefined = undefined
   export let onChange: ((value: Ref<Employee>) => void) | undefined = undefined
```

Comment:
@BykhovDenis Why we have onChange in Presenter? It seems I've missed when it was added.

Reply:
@haiodo It allow us to change value in table or list

Reply:
@BykhovDenis it should not be presenter, it should be editor.